README.md

```diff
@@ -66,139 +69,222 @@ or
   :load-path "/path/to/lib")
 ```
 
-## 📇 Commands
+## 🖥 Usage
+
+### 📇 Commands
 
-| Commands                   | Description                                                                 |
+The following are the functions provided by `ts-fold-mode`
+| Commands | Description |
 |----------------------------|-----------------------------------------------------------------------------|
-| `ts-fold-close`            | fold the current syntax node.                                               |
-| `ts-fold-open`             | open all folds inside the current syntax node.                              |
+| `ts-fold-close` | fold the current syntax node. |
+| `ts-fold-open` | open all folds inside the current syntax node. |
 | `ts-fold-open-recursively` | open the outmost fold of the current syntax node. Keep the sub-folds close. |
-| `ts-fold-close-all`        | close all foldable syntax nodes in the current buffer.                      |
-| `ts-fold-open-all`         | open all folded syntax nodes in the current buffer.                         |
-| `ts-fold-toggle`           | toggle the syntax node at `point'.                                          |
+| `ts-fold-close-all` | close all foldable syntax nodes in the current buffer. |
+| `ts-fold-open-all` | open all folded syntax nodes in the current buffer. |
+| `ts-fold-toggle` | toggle the syntax node at `point'. |
```

Comment:
I think it's better if we have the table aligned.

Reply:
oh, ew, totally agree. I had prettier running on save but it seems to not like tables. Will disable and fix up.